test/acl.test.js

```diff
@@ -75,23 +77,22 @@ describe('security scopes', function() {
             model: 'testModel', property: 'name',
             accessType: ACL.WRITE, permission: ACL.DENY,
           }, function(err, resource) {
-          // console.log(resource);
-            Scope.checkPermission('testModelScope', 'testModel', ACL.ALL, ACL.ALL,
-              function(err, perm) {
-                assert(perm.permission === ACL.DENY); // because name.WRITE == DENY
-              });
-            Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.ALL,
-              function(err, perm) {
-                assert(perm.permission === ACL.DENY); // because name.WRITE == DENY
-              });
-            Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.READ,
-              function(err, perm) {
-                assert(perm.permission === ACL.ALLOW);
-              });
-            Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.WRITE,
-              function(err, perm) {
-                assert(perm.permission === ACL.DENY);
-              });
+            async.parallel([
+              clbk => Scope.checkPermission('testModelScope', 'testModel', ACL.ALL, ACL.ALL, clbk),
+              clbk => Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.ALL, clbk),
+              clbk => Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.READ, clbk),
+              clbk => Scope.checkPermission('testModelScope', 'testModel', 'name', ACL.WRITE, clbk),
+            ], (err, perms) => {
+              assert(!err);
```

Comment:
Let's simplify this code and simply forward the error to `done` callback. It will also provide a more helpful error message.

```js
if (err) return done(err);
```